### DISCUSSION TITLE
Convert ActiveModel to Model

### DISCUSSION THREAD

**Discussion by Qwertie-** [SeaQL/sea-orm#929] [Q&A] [answered]
From the [update docs](https://www.sea-ql.org/SeaORM/docs/basic-crud/update/) its shown that a Model can be converted to ActiveModel with `into()`. But I can't see any way to go the other way, converting an ActiveModel to Model. Is this possible to achieve without running another query to find the record as a Model?

**Comment by billy1624** [comment, marked as answer]
Hey @Qwertie-, welcome! Please check the related issues / PRs and feel free to join our discussions :)
- https://github.com/SeaQL/sea-orm/issues/606
- https://github.com/SeaQL/sea-orm/pull/725
